docs/ref/booking/notes.rst

```diff
@@ -1,14 +1,14 @@
 booking/notes
 --------------
-This is a call to your Checkfront account that pulls all the notes from all of your bookings within a certain time frame. By default this will pull all notes from all your bookings in the system. 
+This is a call to your Checkfront account that pulls all the notes from all of your bookings. This defaults to pulling all the notes from your bookings for the past thirty days. 
```

Comment:
should be 
This defaults to pulling all **of** the notes from your bookings for the past thirty days.